Arrays_and_Cycles/js-arrays-1.js

```diff
@@ -0,0 +1,18 @@
+/* Дан массив состоящий из названий фильмов, выполните перебор
+массива с выводом в консоль названия каждого фильма*/
+let movies = [
+  "Marvel's The Avengers",
+  "Iron Man",
+  "Thor",
+  "Captain America: The Winter Soldier",
+  "Doctor Strange",
+  "Avengers: Infinity War",
+  "Eternals",
+  "Black Widow",
+];
+function outputMovies() {
+  for (let i = 0; i < movies.length; i++) {
+    console.log(movies[i]);
+  }
+}
```

Comment:
Tip: you can use `forEach()` method too. Read here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach